DQM/L1TMonitor/python/L1TStage2uGMT_cfi.py

```diff
@@ -117,3 +117,41 @@
     verbose = cms.untracked.bool(False),
 )
 
+# The five modules below compare the primary unpacked uGMT muon collection to goes to uGT board 0
```

Comment:
hi @thomreis @rekovic - this file needs to be renamed as a "cff" not a "cfi" (which should define one module). Please have a look in a near-term followup pr. thanks